src/libs/actions/Report.js

```diff
@@ -904,21 +919,27 @@ function addAction(reportID, text, file) {
  * network layer handle the delayed write.
  *
  * @param {Number} reportID
- * @param {Number} sequenceNumber
+ * @param {Number} [sequenceNumber]
  */
 function updateLastReadActionID(reportID, sequenceNumber) {
-    const currentMaxSequenceNumber = reportMaxSequenceNumbers[reportID];
-    if (sequenceNumber < currentMaxSequenceNumber) {
-        return;
-    }
+    // If we are not specifying the sequence number, let's set it to the max one available
+    const lastReadSequenceNumber = sequenceNumber || reportMaxSequenceNumbers[reportID];
 
-    setLocalLastRead(reportID, sequenceNumber);
+    setLocalLastRead(reportID, lastReadSequenceNumber);
+
+    // If we are specifying the last read sequence number, we are manually marking a comment as unread, so
```

Comment:
```suggestion
    // If we are specifying sequenceNumber, we are manually marking a comment as unread, so
```

Reply:
I have been trying to emphasize that what we are passing is the `last read sequence`. I think this is relevant because:
- We store the last read sequence in the NVP (and that's what we use down the line to determine the number of unread actions)
- BUT, we place the new marker above the `first unread sequence` (not the last read sequence)